web/src/lib/store.ts

```diff
@@ -0,0 +1,34 @@
+import { browser } from '$app/environment';
+import type { Writable } from 'svelte/store';
+import { get, writable } from 'svelte/store';
+
+/**
+ * Create a store that caches data to localstorage
+ */
+export const localWritable =
```

Comment:
for the smooth brains, why have something like this instead of reading and writing straight to/from localstorage?

Reply:
For reactivity really. This just creates a hook to the reactivity engine to read from and write to localstorage.